### PR TITLE
Feedback updates

### PR DESCRIPTION
**Pull request by a-a-ron**
Hey, @mattdavis0351. I made a couple minor changes to the exercise

- Updated the troubleshoot message from .gitignore to dependabot
- Updated the README with an exercise specific grading results image
- Made some minor changes to the feedback messages

### CODE REVIEW

**Comment by mattdavis0351** [comment]
I'm looking at this on my phone so it's a little hard to tell, but you don't want to edit anything in **dist** by hand. 

The process is:
- make changes in `lib/gradeLearner`
- run `npm run build` from the terminal (this updates stuff in `dist/index.js`)
- push changes 

So if those are the steps you took then all looks good 😊😊

**Comment by mattdavis0351** [comment]
@a-a-ron thanks for the extra set of eyes

**Comment by a-a-ron** [comment]
@mattdavis0351 thanks for the reminder 🤦 .  I reverted that commit and ran `npm run build` from `dependabot-pr-action` and ran into the following:

Screen Shot 2021-06-15 at 11 53 41 AM

From what I found, `ENOENT` generally means the script was unable to find the target executable path that was specified

File: .github/workflows/grading.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: troubleshooting steps
-        run: echo "If you're stuck, this documentation may be helpful; https://git-scm.com/docs/gitignore"
+        run: echo "If you're stuck, this documentation may be helpful; https://docs.github.com/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/configuring-dependabot-security-updates"
```
Comment (hectorsector) [inline .github/workflows/grading.yml:31]: Thanks for this, @a-a-ron ❤️

**Comment by mattdavis0351** [comment]
@a-a-ron I gave you bad instructions.  You need to first install dependencies 


First, make sure you have [NCC Installed on your computer](https://github.com/vercel/ncc)


Then, from the root of the repo:

- `cd .github/actions/dependabot-pr-action`
- `npm install`
- `npm run build`
- push changes

You got the error because **ncc is not found** meaning you don't have the ability to build 😄

**Comment by mattdavis0351** [comment]
@a-a-ron I went ahead and compiled and pushed the changes.  I'll make a note of NCC being a blocker for codespaces and see what we can do about adding a dev container that has it as a part of the repo template tool I'm working on.

This should be ready to 🚢

**Pull request merged**